site/data/data.json

```diff
@@ -4238,6 +4374,20 @@
       "The Cleveland Bagel Co."
     ]
   },
+  {
+    "name": "Cleveland Bagel Company",
```

Comment:
Should be alias for cleveland bagel above.

Reply:
adding